pages/common/git-rm.md

```diff
@@ -0,0 +1,15 @@
+# git rm
+
+> Remove files from the working tree and from the index
+
+- Remove file and add its deletion to next commit
+
+`git rm {{file}}`
+
+- Remove file from all previous commits but keep it locally
```

Comment:
>>  Does this actually rewrite history by removing the file from previous commits??

That is write. I don't know how i committed this. Probably copy-pasted from somewhere and forgot to check before actually committing.